test/govuk_component/metadata_test.rb

```diff
@@ -59,4 +59,48 @@ def component_name
     assert_definition('Part of:', 'this and that')
     assert_definition('Related topics:', 'a, b, and c')
   end
+
+  test "rendering of long metadata into wrapping for Javascript interaction" do
```

Comment:
This test name reads a little oddly.
Maybe: "long lists of metadata are hidden behind toggle"